test/e2e/managementstate_test.go

```diff
@@ -131,3 +131,56 @@ func TestManagementStateRemoved(t *testing.T) {
 		t.Fatalf("deployment is expected to be removed, got %v %v", d, err)
 	}
 }
+
+func TestRemovedToManagedTransition(t *testing.T) {
```

Comment:
@ricardomaraschini I suspect this test is pushing the e2e over the 30 minute timeout in the makefile target. Origin uses 2 hours as an acceptable timeout for e2e tests - I think we can do the same given that these are operator tests that cause rollouts to occur.

Reply:
Increased to 2h.